src/main/java/seedu/address/model/lesson/Time.java

```diff
@@ -0,0 +1,75 @@
+package seedu.address.model.lesson;
```

Comment:
Actually, I think we can make the Time, Date and DateTime class inherit or implement a common interface. Maybe I'll refactor this later.